### PR TITLE
Added installationOptions information for enterprise app for windows platform

### PR DESCRIPTION
**Pull request by twitharshil**

### CODE REVIEW

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 1]
```diff
@@ -117,6 +119,24 @@ The `msiexec` tool can be configured to output debug log information about the i
 msiexec /i path\to\package.msi /l*v C:\log.txt
 ```
 
+### MSI InstallationOptions
```
Comment (RaisinTen) [inline src/pages/docs/administration/managing-enterprise-deployment.md:122]:
```suggestion
### MSI installation Options
```

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 2]
```diff
@@ -117,6 +119,24 @@ The `msiexec` tool can be configured to output debug log information about the i
...
+
+Settings can be changed during the installation time for MSI application.
+
+For example, if your app defines a `MY_OPTION` installation option, you can set a custom value:
+
+``` shell
+msiexec /i path/to/package.msi MY_OPTION='hello'
```
Comment (RaisinTen) [inline src/pages/docs/administration/managing-enterprise-deployment.md:129]:
```suggestion
msiexec /i path\to\package.msi MY_OPTION='hello'
```

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 3]
```diff
@@ -117,6 +119,24 @@ The `msiexec` tool can be configured to output debug log information about the i
...
+```
+
+Installing a MSI with some of its installation options looks like this:
+
+``` shell
+msiexec /i path/to/package.msi MY_OPTION='hello'
+msiexec /i path/to/package.msi MY_OPTION=YES
+msiexec /i path/to/package.msi MY_OPTION=7
```
Comment (RaisinTen) [inline src/pages/docs/administration/managing-enterprise-deployment.md:137]:
```suggestion
msiexec /i path\to\package.msi MY_STRING_OPTION=hello MY_INTEGER_OPTION=7
```
I think `msiexec` treats booleans the same way as integers.

**Review by RaisinTen — CHANGES_REQUESTED** [review]

File: src/pages/docs/administration/managing-enterprise-deployment.md (hunk 2 above)
Comment (twitharshil) [reply src/pages/docs/administration/managing-enterprise-deployment.md:129]: @RaisinTen Are you sure we don't need quotes around string values ?

Comment (RaisinTen) [reply src/pages/docs/administration/managing-enterprise-deployment.md:129]: Yea, see how we pass the `INSTALLDIR` property in https://bitbucket.org/postmanlabs/starship/src/master/docs/windows/using-msi.markdown. You may double-check it locally though. ;)

I _think_ we need the quotes (maybe both `'`/`"` will do) for strings with spaces.

Comment (twitharshil) [reply src/pages/docs/administration/managing-enterprise-deployment.md:129]: Yup, so it works fine if we add quotes I've tested that

File: src/pages/docs/administration/managing-enterprise-deployment.md [hunk 4]
```diff
@@ -117,6 +119,22 @@ The `msiexec` tool can be configured to output debug log information about the i
...
 ```
 
+### MSI installation options
+
+Settings can be changed during the installation time for MSI application.
+
+For example, if your app defines a `MY_OPTION` installation option, you can set a custom value:
+
+``` shell
+msiexec /i path/to/package.msi MY_OPTION='hello'
```
Comment (RaisinTen) [inline src/pages/docs/administration/managing-enterprise-deployment.md:129]:
```suggestion
msiexec /i path\to\package.msi MY_OPTION='hello'
```
we still need the `\`s on windows

**Review by RaisinTen — APPROVED** [review]
LGTM

**Review by jkonrath-postman — APPROVED** [review]
This branch was stale, so I pulled the latest develop into it. Sorry for the noise there.

I made two small changes in managing-enterprise-deployment.md - otherwise, looks good. @twitharshil let me know if you want to merge or have more changes.

**Pull request merged**